app/models/constants/case_log.rb

```diff
@@ -144,6 +144,11 @@ module Constants::CaseLog
     "Yes" => 1,
   }.freeze
 
+  POLAR3 = {
```

Comment:
What's the purpose/distinction between of `POLAR`, `POLAR2` and `POLAR3`? Shouldn't they all be the same (a mapping of `0: "No" and 1: "Yes"`?

Reply:
Well, POLAR and POLAR2 are different because depending on the field we save answers differently in the db (based on the data dictionary, which is a bit inconsistent)

POLAR3 is for `nocharge` column where the question sort of gets asked inversely ("Does the household pay rent or other charges for the accommodation?"), so if answered "No" it would mean that `nocharge` is true and therefore based on the data dictionary should be saved as 1 in the db. 

Would it make sense to instead add a new field for this question and consider `nocharge` as a derived variable?

Reply:
Ah right, if the inconsistency is there already in the data dictionary then yeah I guess we need to keep it :disappointed: 

`nocharge` as a derived variable seems more intuitive to me if we're not asking it directly but either way works.

Reply:
changed that now as I do agree it's probably more intuitive. Could you give that a glance?